pinot/connectionFactory_test.go

```diff
@@ -35,3 +36,17 @@ func TestPinotClients(t *testing.T) {
 	assert.NotNil(t, err)
 	assert.True(t, strings.Contains(err.Error(), "please specify"))
 }
+
+func TestPinotWithHttpTImeout(t *testing.T) {
+	pinotClient , err := NewWithConfig(&ClientConfig{
+		BrokerList: []string{"www.google.com:81"},
+		HTTPTimeout: 1 * time.Second,
+	})
+	assert.Nil(t,err)
+	start := time.Now()
+	_ , err = pinotClient.ExecuteSQL("testTable","select * from testTable");
+	end := time.Since(start)
+	assert.NotNil(t,err)
+	diff := int(end.Seconds())
+	assert.Equal(t,1,diff)
```

Comment:
How accurate is this time check? Shall we make a range check like smaller than 2?

Reply:
Updated. Ideally "int" conversion should round it off. But for safer side, I added a less than 2 sec assertion